shuup/core/models/_products.py

```diff
@@ -90,8 +90,8 @@ class ShippingMode(Enum):
     SHIPPED = 1
 
     class Labels:
-        NOT_SHIPPED = _('not shipped')
-        SHIPPED = _('shipped')
+        NOT_SHIPPED = _('not shipped(not deliverable)')
+        SHIPPED = _('shipped(deliverable)')
```

Comment:
Change these to `not shipped (non-deliverable)` and `shipped (deliverable)`. The most important thing, however, is the missing space between the `d` and `(`.

Also, if the goal is to make these options simpler to understand, consider changing the `help_text` where these are being used.